packages/tasks/src/model-libraries.ts

```diff
@@ -1,52 +1,333 @@
+import * as snippets from "./library-ui-elements";
```

Comment:
Let's rename the file to make it more explicit it only contains snippets maybe

Reply:
a4c7c52a9d179252ed7d5c2657a50fe11ed6ea2c